src/parser.rs

```diff
@@ -687,32 +680,28 @@ mod tests {
     parse_statement_matcher_test_case! {
         name: let_binary_divide,
         input: "let x = a / b;",
-        matcher: LetStatementMatcher{
-            identifier: "x".to_string(),
-            mutable: false,
-            matcher: Box::new(BinaryOperatorExpressionMatcher{
-                left: Box::new(IdentifierMatcher{identifier: "a".to_string()}),
-                right: Box::new(IdentifierMatcher{identifier: "b".to_string()}),
-                operator: OpName::Divide,
-            })
-        },
+        matcher: match_let_stmt!(
```

Comment:
why abbreviate to `stmt` here and throughout?

Reply:
Well-spotted it's inconsistent and needs fixing.

Reply:
Fixed.